docs/adrs/0080-use-flipt-feature-flags.md

```diff
@@ -0,0 +1,106 @@
+
+Scenario 2: Changing Authentication
+
+After we have been live for several months, the customer requires us to move from using login.gov to authenticate users to Okta. Writing and testing the code is no problem. But when we deploy it, every user now has to set up a new authentication account. Many of those users will be in the middle of moving to their next duty station when we spring this on them. Users are constantly moving, there will not be a break in the schedule where we can do the changeover.
+
+Scenario 3: Prime Freeze
+
+For 30 days before we go live, the Prime (HSA) requires a code freeze for testing. We do not want to stop deploying for 30 days, and we really don't want to stop development for 30 days. Feature flags would allow us to build (and maybe even deploy) code during the freeze and remain productive.
+
+## Considered Alternatives
+
+* Option #0: Do Nothing. Stay the course, don't make any changes, figure out what to do when problems arise
+* Option #1: LaunchDarkly: This is the industry standard feature flag platform.
+* Option #2: Unleash: Open source, can be self-hosted
+* Option #3: AWS AppConfig: we're already using lots of AWS, what's a little more?
+* Option #4: Roll our own
+* Option #5: Flipt: Open source, self hosted, supports file based config
+
+## Decision Outcome
+
+Chosen Alternative: Option #5: Flipt
+
+LaunchDarkly is the industry standard, but there is no room for it in the budget. Unleash has similar features to AWS AppConfig, but requires more setup and maintenance effort when self-hosted (e.g. it requires a PostgreSQL database). AWS AppConfig is really a thin shim over config files and doesn't provide much in the way of helping us manage feature groupings.
+
+Rolling our own is a possibility, but one key feature we'd want to think about is how to promote feature flags from one environment to another (e.g. from staging to production). That suggests that tying the feature flag to the environment by e.g. storing the flags in the database isn't ideal as we need to recreate the flag settings in each environment. Combining that with the desire to have feature flags be enabled on a per user basis, it makes the complexity of rolling our own outweigh the relatively simple flipt deployment.
+
+Flipt provides a [filesystem backend](https://www.flipt.io/docs/experimental/filesystem-backends) which would allow a way for us to manage our feature flags using a [gitops](https://about.gitlab.com/topics/gitops/) style process. We can test out our flag configuration in separate environments (e.g. experimental, demo, staging) before rolling out to production. It also allows us to deploy the Flipt service without requiring another stateful system (e.g. no database).
+
+We will have a couple of different options for how we deploy flipt. Examining the options for how flipt is deployed may be its own ADR.
```

Comment:
**suggestion:** If this is not its own ADR, I would recommend a prototype branch. 
If it is its own ADR, then I would remove the word "may be its own" and replace it with "will be its own".

Reply:
I think Drew has a spike PR/branch here: https://github.com/transcom/mymove/pull/10970

Reply:
I've updated the working to "will be". Thank you!